File: src/js/dating.js
 function formattedDate(date){
    var year = date.getFullYear();
    var month = date.getMonth() + 1;
    var day = date.getDate();
    return `${year}-${month < 10 ? "0" + month : month}-${day < 10 ? "0" + day : day}`;
}

function getAgeInYears(birthday){
    return new Date(Date.now() - birthday.getTime()).getFullYear() - 1970;
}

var birthday = new Date(1939, 10, 27);
var age = getAgeInYears(birthday);
console.log(`John Cleese was born in ${formattedDate(birthday)} and is ${age} years old.`);

var date = new Date(Date.parse("1975-11-10T01:25:00"));
console.log(`Parsed date is ${date.toISOString().substr(0, 19)}`);

File: src/py/dating.py
import datetime

def getAgeInYears(birthday):
    today = datetime.date.today()
    return today.year - birthday.year - ((today.month, today.day) < (birthday.month, birthday.day))

birthday = datetime.date(1939, 10, 27)
age = getAgeInYears(birthday)
print("John Clesse was born in ", birthday, "and is ", age, " years old.")

parsedDate = datetime.datetime.strptime("1975-11-10T01:25:00", "%Y-%m-%dT%H:%M:%S")
print("Parsed date is ", parsedDate.strftime('%Y-%m-%dT%H:%M:%S'))